docs/quotes/README.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: docs
+title: Quotes
+permalink: /quotes
+---
+
+# Quotes
+
+Quotes can ultimately be used for PSI element -> String transformations and vice-versa. For example: 
+
+```kotlin
+ifExpression({ true }) { e ->
+     Transform.replace(
+      replacing = e,
+        newDeclaration = """ if $`(condition)` $then $`else` """.`if`
+       )
+}
+```
+
+Below you can find a comprehensive map between Kotlin's PSI element constructs and various quotes in arrow.meta.quotes. If something's missing here, please let us know [here in our issue tracker](https://github.com/arrow-kt/arrow-meta/issues).
+
+## ClassOrObject (quotes.classorobject)
```

Comment:
This table has to go as it will be automatically generated

Reply:
No problem, I can remove this file